lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -83,14 +83,14 @@ class QueryBuilder
     /**
      * The query parameters.
      *
-     * @var mixed[]
+     * @var array<string|int, mixed>
```

Comment:
I don't think that parameter keys can be mixed. It's more of `array<int, mixed>|array<string, mixed>`.